substratest/factory.py

```diff
@@ -451,7 +479,13 @@ def add_composite_traintuple(self, composite_algo, dataset=None, data_samples=No
         self.composite_traintuples.append(spec)
         return spec
 
-    def add_testtuple(self, objective, traintuple_spec, tag='', metadata=None):
+    def add_testtuple(
+        self,
+        objective,
+        traintuple_spec,
+        tag='',
+        metadata=None,
+    ):
```

Comment:
To ease the review, we should try to keep the code change as light as possible. 
So I think we should avoid unrelated code style change.

Reply:
You are right, I try to avoid it, this one slipped through ! I revert it